src/BotPlutusInterface/Balance.hs

```diff
@@ -146,7 +168,26 @@ balanceTxStep ::
 balanceTxStep minUtxos fees utxos ownPkh tx =
```

Comment:
Could we add fees to directly to Tx and remove the fee from the arguments (just a suggestion)?

Reply:
I actually considered this then got pulled into a meeting, will do!